src/globals/Announcement.ts

```diff
@@ -0,0 +1,17 @@
+import type { GlobalConfig } from 'payload/types'
+
+import { isAdmin } from '../access/isAdmin'
+
+export const Announcement: GlobalConfig = {
```

Comment:
This might be too simple. Thinking ahead we may need more than a single announcement, or need to place specific announcements on only some pages. We may also want to draft announcements as we prepare them or even schedule their release. Should this be a version-enabled collection instead? We don't need to write code for any of these fancy features but should have the plumbing in place when we get there.